wordpress-mobile/WordPress-Android
# Issue/4889 notifications read mark

Fixes #4889 by re-marking the notification as read once the NotificationUpdateService has finished doing its thing, as per [this comment](https://github.com/wordpress-mobile/WordPress-Android/issues/4889#issuecomment-267701075)

To test:

0. Send the Android app to the background (i.e. press menu button on your device)
1. Spawn a WP.com notification, such as by leaving a comment on your site as another user
2. Open the Android notification, which should launch the app to the WP notification
3. Back out to the notifications list

The note should now be marked read.

cc @daniloercoli
